Compiling.md

```diff
@@ -5,23 +5,42 @@ permalink: a/compiling/
 article: true
 ---
 
+A lot changed from ELMAH 1x to ELMAH 2x, why this document is split into two parts.
+
+## ELMAH 2x
+
+You can compile ELMAH from the sources in one of two ways:
```

Comment:
Well, there's two ways to build ELMAH, right? From Visual Studio and MSBuild (through the cmd file). Don't you want to explain the two options here?